requirements.txt

```diff
@@ -1,3 +1,2 @@
-Django>=1.3
-geopy==0.94.2
-python-geohash==0.8.5
+Django==1.8
+-r requirements/requirements-tests.txt
```

Comment:
pretty sure the requirements for non tests should be exactly the same as before.